site/gatsby-site/playwright/e2e-full/apps/checklistForm.spec.ts

```diff
@@ -206,33 +126,14 @@ test.describe('Checklists App Form', () => {
         await expect(page.locator('details')).not.toBeVisible();
     });
 
-    test('Should change sort order of risk items', async ({ page, login }) => {
+    test.skip('Should change sort order of risk items', async ({ page, login }) => {
```

Comment:
Please help me create more seed data for this test.

(This can be done in another pr so we don't stale this one)

Reply:
Ok, opened an issue and assigned to myself https://github.com/responsible-ai-collaborative/aiid/issues/3300.